docs/README.md

```diff
@@ -7,18 +7,4 @@ Welcome to the docs for the LM Evaluation Harness!
 * To learn about the public interface of the library, as well as how to evaluate via the commandline or as integrated into an external library, see the [Interface](https://github.com/EleutherAI/lm-evaluation-harness/blob/big-refactor/docs/user_guide.md)
 * To learn how to add a new library, API, or model type to the library, as well as a quick explainer on the types of ways to evaluate an LM, see the [Model Guide](https://github.com/EleutherAI/lm-evaluation-harness/blob/big-refactor/docs/model_guide.md).
 * For a crash course on adding new tasks to the library, see our [New Task Guide](https://github.com/EleutherAI/lm-evaluation-harness/blob/big-refactor/docs/new_task_guide.md).
-* To learn more about pushing the limits of task configuration that the Eval Harness supports, see the [Advanced Task Guide](https://github.com/EleutherAI/lm-evaluation-harness/blob/big-refactor/docs/advanced_task_guide.md).
-
-## Progress on Revamp
-
-Tracking progress on revamping documentation pages for the refactor of LM-Evaluation-Harness.
-
-### Desired Pages
-
-* [ ] YAML explainer
-  * [ ] Explainer on filters + advanced features
-  * [ ] Walkthrough start-to-finish of adding a new task to codebase
-* [ ] Explaining registries + decorators
-* [ ] model_guide.md for adding new model API
-  * [ ] guide to writing an adapter to new advanced codebase (e.g. NeoX)
-* [ ] Parallelism guide (?)
+* To learn more about pushing the limits of task configuration that the Eval Harness supports, see the [Advanced Task Guide](https://github.com/EleutherAI/lm-evaluation-harness/blob/big-refactor/docs/task_guide.md).
```

Comment:
It's been renamed to `task_guide.md` made we should also denote it as just `Task Guide` and drop the `Advanced`?

Reply:
Good catch! Updated